### DISCUSSION TITLE
When is v8 available in npm?

### DISCUSSION THREAD

**Discussion by thianhooi-torum** [NoahZinsmeister/web3-react#592] [Q&A]
When will v8 released in npm? I am eager to try on the newly added hook.

**Comment by AzzouQ** [comment]
It is already, [see on npmjs](https://www.npmjs.com/package/@web3-react/core/v/8.0.35-beta.0).

To use it, just add manually `"@web3-react/core": "^8.0.35-beta.0"` and others dependencies to your `package.json`. (use the last version specified in `README.md`).

**Comment by niZmosis** [comment]
I know this is pretty old, but incase anyone runs into it, there is a little trick that you can use that will show you a list of every version number to pick from. "yarn add @web3-react/core@^" . The @^ appended to any package will show you a list to pick a version to install.
